Source/ViewableController.swift

```diff
@@ -291,13 +284,26 @@ class ViewableController: UIViewController {
     }
 
     func playAction() {
+        #if os(iOS)
         self.repeatButton.alpha = 0
         self.pauseButton.alpha = 0
         self.playButton.alpha = 0
         self.videoProgressView.alpha = 0
 
         self.videoView.play()
         self.requestToHideOverlayIfNeeded()
+        #else
+            // We use the native video player in Apple TV because it provides us extra functionality that are not
+            // providing in the custom player while at the same time it doesn't decrease the user experience since
+            // it's not expected that the user will drag the video to dismiss it, something that we need to do on iOS.
+            if let url = self.viewable?.url {
+                let controller = AVPlayerViewController(nibName: nil, bundle: nil)
```

Comment:
Sorry if I'm redundant, or explaining something you already knew, but might help to describe what's going on: the ViewerController has a scroll view where it has a list of ViewableControllers, each of this controllers has an image view and a player layer, on iOS if you press the play button the image view and other controls hide and the video starts playing, here instead of hiding the controls we will present the AVPlayerViewController. The reason why we are doing this is in the code comment.